core/parentchain/parentchain-crate/src/primitives.rs

```diff
@@ -22,7 +22,7 @@ use alloc::vec::Vec;
 use codec::{Decode, Encode};
 
 /// Initialization primitives, used to by both service and enclave.
-/// Allows to use E-call for the initialization of different parentchain types.
+/// Allows to use one E-call for the initialization of different parentchain types.
```

Comment:
[optional, nitpicking]
If the emphasis is on the fact only one call is needed for the initialization, how about this?
```suggestion
/// Allows to use a single E-call for the initialization of different parentchain types.
```